reference/interbit-ui-components/contributing.adoc

```diff
@@ -79,6 +79,14 @@ Simple, presentational components should go in the `src/components/UIKit`
 directory. More complex components should be in their own directory within
 `src/components`. Each component must be exported from `src/index.js`.
 
+Presentational components should be dumb components where possible. They:
+
+- Receive data and callbacks as props exclusively.
+- Don't have their own state.
+- If components must have their own UI-related state (e.g. whether a modal is
```

Comment:
+1 Listing requirements. :smile_cat:

Reply:
Nit: Each point is part of a sentence that begins with 'They', and so should not have the first word capitalized.

Reply:
It's corrected now 😄